platform/lib/platform/invites.ex

```diff
@@ -206,7 +206,7 @@ defmodule Platform.Invites do
     else
       # If the code is single use, mark it as used
       if invite.single_use do
-        invite = update_invite(invite, %{active: false})
+        ^invite = update_invite(invite, %{active: false})
```

Comment:
We don't want to pin the variable; if you test this code path, (I believe) this will cause a MatchError. Instead we generally replace the variable name with `_` if we don't care about the response:

```suggestion
        _ = update_invite(invite, %{active: false})
```

In this case, we may actually want to verify that the invite was marked inactive, which we could do by matching as follows (`update_invite` returns a tuple):

```suggestion
        {:ok, _} = update_invite(invite, %{active: false})
```

I mentioned this via email, but the pin operator probably isn't what you want. Here's a good StackOverflow answer that explains what pinning is: https://stackoverflow.com/questions/27971357/what-is-the-pin-operator-for-and-are-elixir-variables-mutable